crates/issue/src/lib.rs

```diff
@@ -351,15 +351,7 @@ impl<T: Config> Pallet<T> {
         // allow anyone to complete issue request
         let requester = issue.requester.clone();
 
-        // only executable before the request has expired
-        ensure!(
-            !ext::btc_relay::has_request_expired::<T>(
-                issue.opentime,
-                issue.btc_height,
-                Self::issue_period().max(issue.period)
-            )?,
-            Error::<T>::CommitPeriodExpired
-        );
+        // note: we allow execution after expiry as long at it has not been cancelled yet
```

Comment:
This comment is wrong right? With this PR we would now allow execution after cancellation?

Reply:
Ah yea that's a leftover from the first iteration of this pr - will remove